BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,205 @@
+[!! Connections (BigCommerce)](../UserInterface/01a_Connections.md)
+
+# Manage BigCommerce connection
+
+*Actindo* features a special installation app in the *BigCommerce App Store*. The *Actindo Core1 DOP connector* app sets up and authorizes a connection automatically. 
+Note, if possible, you should not delete the app in *BigCommerce* after creating the connection. Although deleting will not delete the connection in *Omni-Channel* itself, it will invalidate the access token you created when connecting to the app. If the app was deleted by mistake, either reconnect the app or enter a valid access token manually to enable the connection again.
+
+Note that you can use the *Actindo Core1 DOP connector* app for one connection only. If you have more than one store in *BigCommerce*, you can <!--- Jannik, can oder must?-->create the first connection automatically with the *Actindo Core1 DOP connector* app. For all other stores, you must create the connection manually.
```

Comment:
Das ist nicht ganz korrekt, die App kann nur eine Connection für einen Store in BigCommerce erzeugen. Hat der Kunde mehrere (unterschiedliche) Stores, kann er die App nutzen um mehrere Connections zu erzeugen. Wichtig dabei ist, dass der Storehash unterschiedlich ist. 
Wurde bereits eine Connection mit dem Storehash erzeugt, egal ob durch die App oder manuell, erzeugt die App keine neue Connection, sondern aktualisiert die mit dem passenden Storehash.

Reply:
Done